ajax/libs/can/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "can",
+  "title": "CanJS",
+  "description": "MIT-licensed, client-side, JavaScript framework that makes building rich web applications easy.",
+  "filename": "can.js",
```

Comment:
"filename" should point to the main minified file. I think it would be `"can.min.js"` here.